examples/common/scripts/vtadmin-down.sh

```diff
@@ -2,8 +2,17 @@
 
 source "$(dirname "${BASH_SOURCE[0]:-$0}")/../env.sh"
 
-echo "Stopping vtadmin-web..."
-kill -9 "$(cat "$VTDATAROOT/tmp/vtadmin-web.pid")"
+function stop_vtadmin() {
+  local name="$1"
+  local file="$2"
```

Comment:
```suggestion
  local binary_name="$1"
  local pidfile="$2"
```

Reply:
This looks good to me, and I could hit the "Sign off and commit suggestion" button, but I do not like to do this without actually testing things, and I do not have time to re-test for such a small improvement.